ui/apps/platform/src/index.tsx

```diff
@@ -60,15 +61,17 @@ dispatch(fetchCentralCapabilitiesThunk());
 root.render(
     <Provider store={store}>
         <FeatureFlagsProvider>
-            <ApolloProvider client={apolloClient}>
-                <ConnectedRouter history={history}>
-                    <CompatRouter>
-                        <ErrorBoundary>
-                            <AppPage />
-                        </ErrorBoundary>
-                    </CompatRouter>
-                </ConnectedRouter>
-            </ApolloProvider>
+            <ReduxUserPermissionProvider>
```

Comment:
thought: Since we're moving some state from redux to context and creating these new Providers, I wonder if having a dedicated directory for them would be a good idea. I can imagine something like `providers/`.

Reply:
I was considering this, but wasn't sure what to call it. `providers/` sounds good though. If/when I add a third one, I'll move all these over to the new directory.